admin/jobs/river/river.go

```diff
@@ -149,7 +463,7 @@ func (h *ErrorHandler) HandlePanic(ctx context.Context, job *rivertype.JobRow, p
 	var args string
 	_ = json.Unmarshal(job.EncodedArgs, &args) // ignore parse errors
 	h.logger.Error("Job panicked", zap.Int64("job_id", job.ID), zap.String("kind", job.Kind), zap.String("args", args), zap.Any("panic_val", panicVal), zap.String("trace", trace))
-	// Set the job to be immediately cancelled
+	// Set the job to be immediately cancelled TODO review if we should retry or cancel the job
```

Comment:
Does this need to be handled now?

Reply:
no just wanted to discuss if we should retry jobs in case of panic or just log the panic and mark the job as complete. For now its marking the job as complete, it does so because mostly we will just run into panic again on retry.